src/main/java/dk/alexandra/fresco/lib/helper/builder/AbstractProtocolBuilder.java

```diff
@@ -67,6 +71,10 @@
 	 */
 	@Override
 	public void beginParScope() {
```

Comment:
Wouldn't it be cleaner to do 
`if (parent = null) {
  ParallelProtocolProducer ppp = new ParallelProtocolProducer();
  append(ppp);
  pushProducer(ppp);
} else {
  parent.beginParScopt(); 
}`